File: app/src/main/java/com/codemybrainsout/imageviewer/model/Location.java
package com.codemybrainsout.imageviewer.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by ahulr on 08-06-2017.
 */


public class Location implements Serializable {

    @SerializedName("city")
    @Expose
    private String city;
    @SerializedName("country")
    @Expose
    private String country;

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

}

File: app/src/main/java/com/codemybrainsout/imageviewer/model/Location.kt
package com.codemybrainsout.imageviewer.model

import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

/**
 * Created by ahulr on 08-06-2017.
 */
@Parcelize
data class Location(
    @SerializedName("city")
    @Expose
    var city: String? = null,

    @SerializedName("country")
    @Expose
    var country: String? = null
) : Parcelable